alibi_detect/utils/pydantic_v1/fields/__init__.py

```diff
@@ -0,0 +1,6 @@
+# mypy: disable-error-code="assignment, no-redef"
```

Comment:
Why do you have two files for pydantic v1 imports? `pydantic_v1` and `pydantic_v1.feilds`?

Reply:
They're separate modules, and this was simply the first approach that came to mind. I imagine there could possibly be 1 file, which would have to expose the `fields` module within it, somehow?